pkgs/development/python-modules/pycddl/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, pythonOlder
+, fetchPypi
+, buildPythonPackage
+, rustPlatform
+, pytestCheckHook
+, pytest-benchmark
+, psutil
+, cbor2
+}:
+
+buildPythonPackage rec {
+  pname = "pycddl";
+  version = "0.4.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-w0CGbPeiXyS74HqZXyiXhvaAMUaIj5onwjl9gWKAjqY=";
+  };
+
+  nativeBuildInputs = with rustPlatform; [ maturinBuildHook cargoSetupHook ];
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-g96eeaqN9taPED4u+UKUcoitf5aTGFrW2/TOHoHEVHs=";
+  };
+
+  nativeCheckInputs = [ pytestCheckHook psutil cbor2 pytest-benchmark ];
+  pythonImportsCheck = [ "pycddl" ];
+  pytestFlagsArray = [ "--benchmark-disable" ];
```

```suggestion
  nativeCheckInputs = [ pytestCheckHook psutil cbor2 ];
  pythonImportsCheck = [ "pycddl" ];
```
Please patch benchmark out it instead to avoid the extra dependencies